frontend/src/index.tsx

```diff
@@ -1,14 +1,38 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import { RedirectLoginResult } from '@auth0/auth0-spa-js';
+import { Auth0Provider } from './react-auth0-spa';
 import App from './components/app';
+import config from './auth_config.json';
+import history from './utils/browserHistory';
```

Comment:
would `import { createBrowserHistory as history } from 'history'` work the same?

Reply:
I guess so, it doesn't look wrong. I think it's just common practice to create it in the utils and use it from there.

Pretty much every website that talks about using the history module seems to have a class which imports that method and then exports default. I think it's something to do with history being a global API and should be accessible by all of the app, and by importing it there, all components are using the same instance of the method.

Reply:
It's not the same for the same reason as the other comment:
in `import { createBrowserHistory as history } from 'history'`, history is a function,
in `import history from './utils/browserHistory'`, history is the object that this function returns.

Reply:
Yep that sounds right. Either way - it's not something I came up with, it's what the Auth0 and other docs (such as react-router?) do when using the history package.